app/components/nav-bar.js

```diff
@@ -1,9 +1,14 @@
 import classic from 'ember-classic-decorator';
-import { action } from '@ember/object';
+import { action, computed } from '@ember/object';
 import Component from '@ember/component';
 
 @classic
 export default class NavBar extends Component {
+  @computed('session.currentRouteName')
+  get isEventPageRoute() {
```

Comment:
This returns true if the page is not event page. It should either be named `isNotEventPageRoute` or return opposite of what it does. Please check the function names make sense. Change the name